src/Util/RequestAttributesExtractor.php

```diff
@@ -56,6 +56,12 @@ public static function extractAttributes(Request $request)
             throw new RuntimeException('One of the request attribute "_api_collection_operation_name" or "_api_item_operation_name" must be defined.');
         }
 
+        if (null === $apiRequest = $request->attributes->get('_api_request')) {
+            $result['request'] = true;
+        } else {
+            $result['request'] = (bool) $apiRequest;
```

Comment:
I don't like this cast. It lets through more than what we want to support. We should use an explicit test.